settings.gradle

```diff
@@ -46,6 +46,6 @@ project(':OoyalaSkinSampleApp').projectDir = new File(settingsDir, './OoyalaSkin
 
 // Developer
 //include ':ooyalaSkinSDK'
-//project(':ooyalaSkinSDK').projectDir=new File(settingsDir, '../ios-skin/sdk/android/ooyalaSkinSDK/')
-//include ':ooyalaCoreSDK'
-//project(':ooyalaCoreSDK').projectDir=new File(settingsDir, '../android-sdk/sdk/ooyalaCoreSDK/')
\ No newline at end of file
+//project(':ooyalaSkinSDK').projectDir=new File(settingsDir, '../native-skin/sdk/android/ooyalaSkinSDK/')
+//include ':core'
+//project(':core').projectDir=new File(settingsDir, '../android-sdk/sdk/core/')
```

Comment:
"ooyalaCoreSDK" was quite old old name. Every time you enable this you have to change name "ooyalaCoreSDK" to "core" also along with uncommenting. Just made little easier.